### PR TITLE
January 2023 rke2

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -971,6 +987,20 @@ releases:
...
+    serverArgs: *serverArgs-v1-24-2-rke2r1
+    agentArgs: *agentArgs-v1-24-2-rke2r1
+    charts: &charts-v1-24-10-rke2r1
+      <<: *charts-v1-24-9-rke2r2
+      rancher-vsphere-cpi:
+        repo: rancher-rke2-charts
+        version: 1.4.101
+      rke2-multus:
+        repo: rancher-rke2-charts
+        version: v3.9.3-build2023010901
```
Comment (rancher-max) [inline channels-rke2.yaml:1003]: We should have the harvest CPI/CSI bumps here as well

**Review by rancher-max — CHANGES_REQUESTED** [review]

**Review by rancher-max — APPROVED** [review]

**Review by rancher-max — CHANGES_REQUESTED** [review]
Ah wait, I approved by accident. This is technically correct, but we need this to have the RC versions first so that QA can validate.

**Comment by briandowns** [comment]
> Ah wait, I approved by accident. This is technically correct, but we need this to have the RC versions first so that QA can validate.

Wow, thanks. That slipped right by me somehow. Updated.

**Review by rancher-max — COMMENTED** [review]
Awesome, thank you for the updates! Looks good now and will give final approval after full release 👍

File: channels-rke2.yaml [hunk 2]
```diff
@@ -862,6 +862,22 @@ releases:
...
+      rancher-vsphere-cpi:
+        repo: rancher-rke2-charts
+        version: 1.4.101
+      harvester-cloud-provider:
+        repo: rancher-rke2-charts
+        version: 0.1.1400
+      harvester-csi-driver:
+        repo: rancher-rke2-charts
+        version: 0.1.1500
     featureVersions: *featureVersions-v1
```
Comment (kinarashah) [inline channels-rke2.yaml:882]: Not sure how important is this field, but with this change  `featureVersions: *featureVersions-v1` is now missing for `v1.23.15+rke2r1` so need to add it back

**Review by rancher-max — APPROVED** [review]

**Review by ShylajaDevadiga — APPROVED** [review]

File: data/data.json [hunk 3]
```diff
@@ -28147,7 +28397,507 @@
...
+      "type": "array"
+     },
+     "kubelet-arg": {
+      "type": "array"
+     },
+     "profile": {
+      "nullable": true,
+      "options": [
+       "cis-1.5",
+       "cis-1.6"
```
Comment (cwayne18) [inline data/data.json:28725]: @rancher-max shouldnt this be cis-1.23 here?

**Pull request merged**